scripts/conda-requirements.yaml

```diff
@@ -0,0 +1,120 @@
+channels:
+    - conda-forge
+    - ucb-bar
+    - nodefaults
+
+dependencies:
+    # https://conda-forge.org/feedstock-outputs/
+    #   filterable list of all conda-forge packages
+    # https://conda-forge.org/#contribute
+    #   instructions on adding a recipe
+    # https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/pkg-specs.html#package-match-specifications
+    #   documentation on package_spec syntax for constraining versions
+
+
+    # handy tool for introspecting package relationships and file ownership
+    # see https://github.com/rvalieris/conda-tree
+    - conda-tree
+
+    # bundle FireSim driver with deps into installer shell-script
+    - constructor
+
+    # https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html#using-the-compiler-packages
+    #    for notes on using the conda compilers
+    # elfutils has trouble building with gcc 11 for something that looks like it needs to be fixed upstream
+    # ebl_syscall_abi.c:37:64: error: argument 5 of type 'int *' declared as a pointer [-Werror=array-parameter=]
+    #    37 | ebl_syscall_abi (Ebl *ebl, int *sp, int *pc, int *callno, int *args)
+    #       |                                                           ~~~~~^~~~
+    # In file included from ./../libasm/libasm.h:35,
+    #                  from ./libeblP.h:33,
+    #                  from ebl_syscall_abi.c:33:
+    # ./libebl.h:254:46: note: previously declared as an array 'int[6]'
+    #   254 |                             int *callno, int args[6]);
+    #       |                                          ~~~~^~~~~~~
+    #
+    # pin to gcc=10 until we get that fixed.
+
+    - gcc =10 
+    - gxx =10 
+    - binutils 
+    - conda-gcc-specs
+
+    - dromajo
+    - riscv-tools
+
+    - qemu
+
+    # firemarshal deps
+    - rsync
+    - psutil
+    - doit=0.35.0
+    - gitpython
+    - humanfriendly
+    - e2fsprogs
+    - ctags
+    - bison
+    - flex
+    - expat
+
+    # current version of buildroot won't build with make 4.3 https://github.com/firesim/FireMarshal/issues/236
+    - make!=4.3
+    - bash-completion
+    - sbt
+    - ca-certificates
+    - mosh
+    - gmp
+    - mpfr
+    - mpc
+    - zlib
+    - vim
+    - git 
+    - openjdk
+    - gengetopt
+    - libffi
+    - expat
+    - libusb1
+    - ncurses
+    - cmake
+    - graphviz
+    - expect
+    - dtc
+    - verilator==4.224
+    - screen
+    - elfutils
+    - libdwarf-dev==0.0.0.20190110_28_ga81397fc4
+
+    # clang-format for driver coding style enforcement.
+    - clang-format 
+    - clang-tools
+
+    # python packages
+    # While it is possible to install using pip after creating the
+    # conda environment, pip's dependency resolution can conflict with
+    # conda and create broken environments.  It's best to use the conda
+    # packages so that the environment is consistent
+    - boto3==1.20.21
+    - colorama==0.4.3
+    - argcomplete==1.12.3
```

Comment:
(Low priority) I'm concerned about duplication between `machine-launch-script.sh` and this, specifically listing out needed packages and their versions. They are installed in separate environments but updating things in two places might cause unforeseen bugs later on (although minor).